sample-plugin/pom.xml

```diff
@@ -124,6 +124,10 @@
             <artifactId>gitlab-branch-source</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>ionicons-api</artifactId>
+        </dependency>
```

Comment:
This is going to fail prep in 2.332.x and 2.319.x lines. Maybe you can add it to the 2.346.x and 2.361.x profiles? But then we would also need to introduce a `weekly` profile and make sure `prep.sh` calls that, so a bit bigger change.

Reply:
Looking at `README.md`, I think we can just delete the dependency here, since `ionicons-api` will be picked up transitively via `workflow-cps` on the appropriate LTS lines. Then we just need to tweak `check.groovy` to accept the fact that `ionicons-api` is part of `dependencyManagement` in 2.332.x and 2.319.x but is not used by the plugin.

Reply:
See [1e8f8b6](https://github.com/jenkinsci/bom/pull/1507/commits/1e8f8b6dd83403863e3a6c28536b40ca8797588a).